packages/ui/src/apps/collectionsUi/viewModules/start/Component.js

```diff
@@ -1,126 +1,128 @@
-                  and their curation ("collection management"). Target
-                  collections include zoological, botanical, geological and
-                  paleontological collections, living collections, biodiversity
-                  inventories, observation records, and molecular data. DINA is
-                  primarily intended for large installations servicing the
-                  collection management needs of a country, a region, or a large
-                  institution.`}
-                  </p>
-                </Segment>
-                <Divider horizontal />
-                <Segment padded="very">
-                  <Header as="h3" style={{ fontSize: '2em' }}>
-                    An international partnership for open-source development
-                  </Header>
-                  <p style={{ fontSize: '1.33em' }}>
-                    {`DINA is developed by the DINA consortium, an unincorporated
-                  international partnership among organizations and individuals
-                  for collaborative open-source development. The DINA consortium
-                  was founded in 2014 by six natural history collection
-                  institutions in Europe and North America and is open to
-                  additional members as detailed below. The DINA acronym stands
-                  for "DIgital Information system for NAtural history data", and
-                  has its roots in a Swedish initiative to replace a
-                  heterogeneous collection of unsustainable in-house databases
-                  with a modern, web-based national collection management
-                  system.`}
-                  </p>
-                </Segment>
-              </Grid.Column>
-              <Grid.Column floated="right" width={6}>
-                <Segment padded="very">
-                  <Markdown
-                    fallbackLanguage="en"
-                    textKey="modules.start.changelog"
-                  />
-                </Segment>
-              </Grid.Column>
-            </Grid.Row>
+        <div className="ui center aligned grid">
+          <Grid
+            columns={2}
+            style={{ marginTop: '6.25em', maxWidth: '60em' }}
+            verticalAlign="middle"
+          >
+            <Grid.Column width={10}>
+              <Header
+                as="h1"
+                content={`${moduleTranslate({
+                  capitalize: true,
+                  module: 'start',
+                  textKey: 'welcome',
+                })} ${capitalizeFirstLetterOfEachWord(
+                  username
+                )}! ${moduleTranslate({
+                  capitalize: true,
+                  module: 'start',
+                  textKey: 'startPageText',
+                })}`}
+                style={{
+                  color: '#000000a0',
+                  fontWeight: 200,
+                  lineHeight: '2.5rem',
```

Comment:
I guess we can leave this for now, but it's questionable if it has to be this custom styling. This is not reusable, to do so you would have to add it as a custom style for header. But we are short on time and it would make sense to use one of the existing header variations. but we can leave it as it is.